src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -66,6 +69,7 @@ public RLoginUser login(QUserLogin user) throws Exception {
             // ALSO IN MESSAGE SERVICE: userStreakService.updateUserStreak(dbUser.getEmail());
             try {
                 userStreakService.updateUserStreak(dbUser.getEmail());
+                questService.assignQuests(dbUser.getEmail());
```

Comment:
We are currently calling these methods both in the login service and the AccountController's test auth endpoint. This could be a problem later on if we need to add another method on login and forget to update either one. Suggestion: maybe we can call all these services inside a private method in AccountService, smth called handleUserEntry or initiateUserSession. We can then call this method in the login method and a new account service method that will be called in the test auth endpoint.

Reply:
Yes, I thought the same thing. It would be best if we discussed how to do this and merge both our implementations (stats, streak, quest etc.) to there. Created [LINGUIST-355](https://linguistai.atlassian.net/browse/LINGUIST-355) to handle this.